harpy/assembly.py

```
@@ -0,0 +1,101 @@
+    os.makedirs(f"{output_dir}/logs/snakemake", exist_ok = True)
+    sm_log = snakemake_log(output_dir, asm)
+
+    with open(f"{workflowdir}/config.yaml", "w", encoding="utf-8") as config:
+        config.write(f"workflow: {asm}\n")
+        config.write(f"snakemake_log: {sm_log}\n")
+        config.write(f"output_directory: {output_dir}\n")
+        config.write(f"barcode_tag: {bx_tag.upper()}\n")
+        config.write("spades:\n")
+        config.write(f"    max_memory: {max_memory}\n")
+        if kmer_length == "auto":
+            config.write(f"    k: auto\n")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)